operator-metadata/automation/tests/resources/freshmaker.enabled.test.bundle.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1472 @@
+                # This is because only a persistent volume of a supported storage class type can be resized
+                - storageclasses
+              verbs:
+                - get
+            - apiGroups:
+                - ""
+              resources:
+                # The cluster operator requires "list" permissions to view all nodes in a cluster
+                # The listing is used to determine the node addresses when NodePort access is configured
+                # These addresses are then exposed in the custom resource states
+                # The Kafka Brokers require "get" permissions to view the node they are on
+                # This information is used to generate a Rack ID that is used for High Availability configurations
+                - nodes
+              verbs:
+                - get
+                - list
+          serviceAccountName: strimzi-cluster-operator
+      deployments:
+        - name: amq-streams-cluster-operator-v2.7.0-0
+          spec:
+            replicas: 1
+            selector:
+              matchLabels:
+                name: amq-streams-cluster-operator
+                strimzi.io/kind: cluster-operator
+            template:
+              metadata:
+                annotations:
+                  operator-image: registry.redhat.io/amq-streams/strimzi-rhel9-operator:2.7.0-14
+                  kafka-images: |
+                    3.6.0=registry.redhat.io/amq-streams/kafka-36-rhel9:2.7.0-18
+                    3.7.0=registry.redhat.io/amq-streams/kafka-37-rhel9:2.7.0-13
+                  kafka-previous-image: registry.redhat.io/amq-streams/kafka-36-rhel9:2.7.0-18
+                  kafka-current-image: registry.redhat.io/amq-streams/kafka-37-rhel9:2.7.0-13
+                  bridge-image: registry.redhat.io/amq-streams/bridge-rhel9:2.7.0-14
+                  maven-image:  registry.redhat.io/amq-streams/maven-builder-rhel9:2.7.0-11
+                labels:
+                  name: amq-streams-cluster-operator
+                  strimzi.io/kind: cluster-operator
+                  com.company: Red_Hat
+                  rht.prod_name: Red_Hat_Application_Foundations
+                  rht.prod_ver: 2024.Q2
+                  rht.comp: AMQ_Streams
+                  rht.comp_ver: "2.7"
+                  rht.subcomp: cluster-operator
+                  rht.subcomp_t: infrastructure
+              spec:
+                serviceAccountName: strimzi-cluster-operator
+                volumes:
+                  - name: strimzi-tmp
+                    emptyDir:
+                      medium: Memory
+                  - name: co-config-volume
+                    configMap:
+                      name: strimzi-cluster-operator
+                containers:
+                  - name: strimzi-cluster-operator
+                    image: registry.redhat.io/amq-streams/strimzi-rhel9-operator:2.7.0-14
+                    valueFrom:
```

Comment:
How's this bit working?   I don't understand how the downward API is being used in this case.

Reply:
Removed